vespa-cloud/cord-19-search/pom.xml

```diff
@@ -20,7 +20,18 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <tenant>vespa-team</tenant>                                 <!-- Name from the web console -->
+        <tenant>vespa-team</tenant>                          <!-- Name from the web console -->
         <application>cord-19</application>                   <!-- Name from the web console -->
+        <junit_version>4.11</junit_version>
```

Comment:
in vespa cloud we started using junit 5: https://github.com/vespa-engine/vespa/blob/master/tenant-base/pom.xml#L40

Reply:
Thanks, I'll fix that.